### PR TITLE
Add: Allow deleting bodybag at logistics point

### PR DESCRIPTION
**Pull request by mrNo0b**
- Add: Allow deleting bodybag at logistics point, 
(optional: commented out code will show message to the player)

**Final test:**
- [x] local
- [ ] server

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf [hunk 1]
```diff
@@ -26,6 +26,12 @@ params [
     ["_allowlist", btc_log_obj_created, [[]]]
 ];
 
+// Allow deleting bodybag
+if ((typeOF _veh) isEqualTo "ACE_bodyBagObject") exitWith {
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf:30]:
```suggestion
if ((typeOf _veh) isEqualTo "ACE_bodyBagObject") exitWith { 
```

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf [hunk 2]
```diff
@@ -26,6 +26,12 @@ params [
     ["_allowlist", btc_log_obj_created, [[]]]
 ];
 
+// Allow deleting bodybag
+if ((typeOF _veh) isEqualTo "ACE_bodyBagObject") exitWith { 
+    deleteVehicle _veh; 
+    //["R.I.P"] remoteExecCall ["CBA_fnc_notify", remoteExecutedOwner];
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf:32]:
```suggestion
```
Don't think this is necessary

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf [hunk 3]
```diff
@@ -26,6 +26,12 @@ params [
     ["_allowlist", btc_log_obj_created, [[]]]
 ];
 
+// Allow deleting bodybag
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/server_delete.sqf:29]:
```suggestion
```
The code is self explaining

**Comment by mrNo0b** [comment]
I cant find option to edit the code, maybe its because I deleted the branch?

**Comment by Vdauphin** [comment]
May be ^^'

**Comment by mrNo0b** [comment]
So the only option is to close it now?

**Comment by Vdauphin** [comment]
Yes I think

**Pull request closed without merging**